### PR TITLE
Fix types

### PR DESCRIPTION
**Pull request by blair**
Add some TypeScript improvements.

### CODE REVIEW

File: typings/index.d.ts [hunk 1]
```diff
@@ -275,7 +275,7 @@ declare namespace local {
      *
      * @param {(str: string) => string} [cb]
      */
-    help(cb?: (str: string) => string): void;
+    help(cb?: (str: string) => string): never;
```
Comment (vanesyan) [inline typings/index.d.ts]: @blair `never` type is incorrect for help function, it doesn't throw or failing to terminate.
https://blog.mariusschulz.com/2016/11/18/typescript-2-0-the-never-type

Comment (blair) [reply typings/index.d.ts]: @vanesyan According to https://github.com/tj/commander.js/#helpcb the method "Output help information and exit immediately" so it terminates the process.

Comment (vanesyan) [reply typings/index.d.ts]: Sorry, you're correct. `help` terminates process.

**Comment by vanesyan** [comment]
Thanks!

**Pull request merged**